src/main/java/picard/sam/markduplicates/util/PhysicalLocationForMateCigar.java

```diff
@@ -75,11 +75,9 @@ public boolean equals(Object other) {
 
     @Override
     public int hashCode() {
-        int result = getLibraryId();
+        int result = super.hashCode();
```

Comment:
I think we should start the practice of using `Objects.hash()` instead of computing hashes by hand, e.g. replace this code with
```
return Objects.hash(super.hashCode(), getLibraryId(), getReadGroup());
```